### PR TITLE
Enlighten RFC 1, continued

### PR DESCRIPTION
**Pull request by michielbdejong**
Continuation of https://github.com/interledger/rfcs/pull/364

### CODE REVIEW

File: 0001-interledger-architecture/0001-interledger-architecture.md [hunk 1]
```diff
@@ -18,9 +18,9 @@ The Interledger architecture is heavily inspired by the Internet architecture de
...
+Interledger packets describe amounts of money, which may be settled individually or in aggregate through various kinds of external payment systems collectively referred to as *ledgers*. Ledgers can include blockchains, but also banks, peer-to-peer payment schemes, automated clearing house (ACH), mobile money institutions and even central-bank operated real-time gross settlement (RTGS) systems among many other types. Some ledgers support very efficient settlement, while others support only slower forms. In general, the faster and cheaper the ledger, the more often an account is settled. See [IL-RFC 22](../0022-hashed-timelock-agreements/0022-hashed-timelock-agreements.md) for more information on different ledger types and settlement strategies. Settlement does not occur between the sender and receiver directly. Each connector settles only with its direct neighbors.
```
Comment (michielbdejong) [inline 0001-interledger-architecture/0001-interledger-architecture.md]: Changes in this line are https://github.com/interledger/rfcs/pull/364#discussion_r176351698

File: 0001-interledger-architecture/0001-interledger-architecture.md [hunk 2]
```diff
@@ -18,9 +18,9 @@ The Interledger architecture is heavily inspired by the Internet architecture de
 
 ### Connectors
 
-A *connector* is a system that provides the service of forwarding Interledger packets towards the destination provided in the packet. Packets are forwarded across connections between senders, connectors and receivers, called *accounts*.
+A *connector* is a system that provides the service of forwarding Interledger packets towards the destination provided in the packet. Packets are forwarded across connections (called *accounts*) between senders, connectors and receivers.
```
Comment (michielbdejong) [inline 0001-interledger-architecture/0001-interledger-architecture.md]: As described in https://github.com/interledger/rfcs/pull/364#discussion_r176349987

File: 0001-interledger-architecture/0001-interledger-architecture.md [hunk 3]
```diff
@@ -61,21 +61,21 @@ The "Interledger protocol suite" can be used among any peered group of connector
...
+When forwarding a `prepare` packet from the sender-side account to the receiver-side account, each connector will reduce the timeout by a fixed amount. This provides a fixed time window to pass back the `fulfill` packet from the receiver-side account to the sender-side account later. A connector must have a fair chance to pass the `fulfill` packet back before the timeout on the sender-side account is reached, even if the `fulfill` packet was received on the receiver-side account at the last possible moment. Otherwise, it would be obligated to pay out on the receiver-side account with no obligation on their counterparty on the sender-side account to pay them back.
```
Comment (michielbdejong) [inline 0001-interledger-architecture/0001-interledger-architecture.md]: See https://github.com/interledger/rfcs/pull/364#discussion_r159137859

**Comment by michielbdejong** [comment]
@emschwartz or @justmoon, can this be merged along with Stefan's original changes from #364?

File: 0001-interledger-architecture/0001-interledger-architecture.md [hunk 4]
```diff
@@ -45,13 +45,13 @@ Senders typically choose a connector and send all their traffic through this con
 
 A *receiver* is a system which accepts incoming Interledger packets and provides a cryptographic receipt for each packet which is returned to the sender along the same path. For more information on the cryptographic exchange, see [Payment Flow](#payment-flow).
 
-**Interledger ensures that senders' funds are safe throughout an multi-hop payment and cannot be stolen by faulty or malicious connectors (see [Interledger Security](#interledger-security)).**
+**Interledger ensures that senders' funds are safe throughout an multi-hop payment and cannot be stolen by faulty or malicious intermediaries (see [Interledger Security](#interledger-security)).**
```
Comment (emschwartz) [inline 0001-interledger-architecture/0001-interledger-architecture.md]: typo: "a**n** multi-hop"

**Pull request merged**